concepticondata/conceptlists/Reinhard-1970-337.tsv

```diff
@@ -0,0 +1,338 @@
+Kusunda-1972-337-80	80	play (1st)	1413	PLAY
+Kusunda-1972-337-81	81	move (1st)	2874	MOVE (ONESELF)
+Kusunda-1972-337-82	82	wife's father		
+Kusunda-1972-337-83	83	flower	239	FLOWER
+Kusunda-1972-337-84	84	inside	1606	INSIDE
+Kusunda-1972-337-85	85	language	1307	LANGUAGE
+Kusunda-1972-337-86	86	meat	634	MEAT
+Kusunda-1972-337-87	87	breathe (1st)	1407	BREATHE
+Kusunda-1972-337-88	88	male sexorgan		
+Kusunda-1972-337-89	89	he/ person	1211	HE
+Kusunda-1972-337-90	90	wood 	2141	TREE OR WOOD
+Kusunda-1972-337-91	91	female sex organ		
+Kusunda-1972-337-92	92	wife's mother		
+Kusunda-1972-337-93	93	a kind of tree		
+Kusunda-1972-337-94	94	thick	1244	THICK
+Kusunda-1972-337-95	95	One-sided drum	908	DRUM
+Kusunda-1972-337-96	96	forest	420	FOREST
+Kusunda-1972-337-97	97	he, she	2642	HE OR SHE
+Kusunda-1972-337-98	98	they	817	THEY
+Kusunda-1972-337-99	99	seed	714	SEED
+Kusunda-1972-337-100	100	sweet/ sap	717	SWEET
+Kusunda-1972-337-101	101	skin	763	SKIN
+Kusunda-1972-337-102	102	feather	1201	FEATHER
+Kusunda-1972-337-103	103	egg	744	EGG
+Kusunda-1972-337-104	104	unpleasant	26	UNPLEASANT
+Kusunda-1972-337-105	105	this morning	1339	MORNING
+Kusunda-1972-337-106	106	tomorrow	1329	TOMORROW
+Kusunda-1972-337-107	107	mother's older brother		
+Kusunda-1972-337-108	108	tobacco used in puja	974	TOBACCO
+Kusunda-1972-337-109	109	monkey	1350	MONKEY
+Kusunda-1972-337-110	110	drink (imperative)	1401	DRINK
+Kusunda-1972-337-111	111	bone	1394	BONE
+Kusunda-1972-337-112	112	burn (1st)	2102	BURN
+Kusunda-1972-337-113	113	green	1425	GREEN
+Kusunda-1972-337-114	114	which	727	WHICH
+Kusunda-1972-337-115	115	small tiger, leopard	1139	LEOPARD
+Kusunda-1972-337-116	116	leaf	628	LEAF
+Kusunda-1972-337-117	117	net bag (see also amji)	293	NETBAG
+Kusunda-1972-337-118	118	where	1237	WHERE
+Kusunda-1972-337-119	119	neck	1333	NECK
+Kusunda-1972-337-120	120	sit (imperative)	1416	SIT
+Kusunda-1972-337-121	121	wither		
+Kusunda-1972-337-122	122	count (1st)	1420	COUNT
+Kusunda-1972-337-123	123	many,much	1198	MANY
+Kusunda-1972-337-124	124	salt	1274	SALT
+Kusunda-1972-337-125	125	look (1st)	1819	LOOK
+Kusunda-1972-337-126	126	small, young	1207	YOUNG
+Kusunda-1972-337-127	127	far	1406	FAR
+Kusunda-1972-337-128	128	spit, throw (1st)	1456	THROW
+Kusunda-1972-337-129	129	dirty	1230	DIRTY
+Kusunda-1972-337-130	130	tall	711	TALL
+Kusunda-1972-337-131	131	now	1376	NOW
+Kusunda-1972-337-132	132	today	1283	TODAY
+Kusunda-1972-337-133	133	fever	1534	FEVER
+Kusunda-1972-337-134	134	shim		
+Kusunda-1972-337-135	135	sun	1343	SUN
+Kusunda-1972-337-136	136	night	1233	NIGHT
+Kusunda-1972-337-137	137	near here	1942	NEAR
+Kusunda-1972-337-138	138	horn	1393	HORN (ANATOMY)
+Kusunda-1972-337-139	139	a kind of  vine	343	VINE
+Kusunda-1972-337-140	140	near	1942	NEAR
+Kusunda-1972-337-141	141	there	1937	THERE
+Kusunda-1972-337-142	142	root/tuber	670	ROOT
+Kusunda-1972-337-143	143	presently		
+Kusunda-1972-337-144	144	fire	221	FIRE
+Kusunda-1972-337-145	145	cry (1st)	1839	CRY
+Kusunda-1972-337-146	146	sashes		
```

Comment:
maybe error when typing off?

Reply:
yes, it is actually "ashes", editing now